src/Services/AlbumService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Services;
+
+use App\Entity\Album;
+use App\Repository\TravelRepository;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+
+class AlbumService
+{
+    // ------------------------ >
+
+    public function __construct(
+        private EntityManagerInterface $em,
+        private TravelRepository $travelRepository
+    ) {
+    }
+
+    // ------------------------ >
+
+    /**
+     * @throws Exception
+     */
+    public function create(
+        string $title,
+        string $description,
+        int $travelId
```

Comment:
Du coup si tu récupère le travel dans le controller, tu peux passer ici Un `
```suggestion
        Travel $travel
```